guide.cabal

```diff
@@ -182,6 +183,10 @@ library
                      , DeriveGeneric
                      , TypeApplications
                      , NoImplicitPrelude
+                     , GeneralizedNewtypeDeriving
```

Comment:
Let's not turn on this extension for all modules – it implicitly changes the behavior of `deriving` for newtypes. Other extensions are good to go.